shopware/administration/6.4/bin/build-administration.sh

```diff
@@ -50,5 +50,8 @@ else
     echo "Cannot check extensions for required npm installations as jq is not installed"
 fi
 
-(cd "${ADMIN_ROOT}"/Resources/app/administration && npm install --no-audit --prefer-offline && npm run build)
+# Dump entity schema
+"${BIN_TOOL}" -e prod framework:schema -s 'entity-schema' src/Administration/Resources/app/administration/test/_mocks_/entity-schema.json
```

Comment:
The target folder is not there, we have to create it (excluded by gitattributes)
https://github.com/shopware/platform/blob/trunk/src/Administration/.gitattributes#L5